TWT/templates/timathon/codejam_listView.html

```diff
@@ -20,7 +20,7 @@ <h6 class="text-muted card-subtitle mb-2" style="text-align: center;">{{challeng
                 <p class="card-text" style="text-align: center;">{{ challenge.challenge.short_desc }}</p>
                 {% if challenge.challenge.youtube_video_link %}
                     <div class="text-center">
-                        <a type="button" class="btn btn-info" href="{{ challenge.challenge.youtube_video_link }}" target="_blank">Youtube Video</a>
+                        <iframe width="90%" height="554" src="{{ challenge.challenge.get_video_embed_link }}" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
i don't think `90%` is a good "responsive" solution

Reply:
:hmmGe:

Reply:
what do u suggest i do?
media queries ?